geo/src/algorithm/mod.rs

```diff
@@ -232,6 +232,11 @@ pub mod triangulate_earcut;
 #[cfg(feature = "earcutr")]
 pub use triangulate_earcut::TriangulateEarcut;
 
+/// # Vector Operations for 2D coordinates
+/// 
+mod vector_ops;
+pub(crate) use vector_ops::Vector2DOps;
```

Comment:
Is there a reason for this not to be public?

Reply:
Hi @michaelkirk no stress about the delay, thankyou for your time :)

No problems making it public. I was thinking of this as an internal tool, but some crate users may find it useful. 

One benefit of making it private is to reduce the complexity of the API exposed to the user. For example, a user might want to find the length of a `Line`, but miss the `EuclideanLength` trait in the docs, and mistakenly assume they should use `Vector2DOps::magnitude` on the `start` and `end`.

On the other hand making it private means 'cargo doc' may hide it away and make it harder for other crate contributors to use it. The user also misses out on some useful functionality.

I think changing it to be public is probably the way to go, and follows the established pattern in `algorithm/mod.rs`?

Reply:
:+1: I'm in favor of making them public and treating them like all our other algorithm modules. They seem like generally useful methods.

Reply:
Ok sounds like a plan to me :)